task-buffer/task-buffer.ts

```diff
@@ -8,7 +8,7 @@ import {
   type Task,
   useScope,
   withResolvers,
-} from "npm:effection@4.0.0-alpha.3";
+} from "npm:effection@4.0.0-alpha.4";
```

Comment:
We should probably as a practice be using ranged versions here, that way, the generated packages will be usable with a range of effection values.